.github/workflows/core.yml

```diff
@@ -51,6 +52,11 @@ jobs:
         os:
           - ubuntu-18.04
           - windows-latest
+        include:
+          - python-version: '3.8'
```

Comment:
Do we need this restriction?  Or more specifically, does our coverage change in different interpreters (due to dependencies, etc)?

Reply:
My guess is that there should be very little if any difference, but we can check it: after this is merged, I'll open another PR removing this restriction, and the coverage diff with the base commit should give us this information.